add sorting nested json example (#209)
xataio/mdx-docs: 040-SDK/101-get.mdx

Note that random sorting does not apply to a specific column, hence the special column name `"*"`.

> It is possible to sort on nested JSON fields with SQL using the `->` operator and quoting the json key.

Reviewer: ```suggestion
You can sort nested JSON fields in SQL by using the `->` operator and enclosing the JSON key in quotes.
```

<TabbedCode tabs={['TypeScript','Python','SQL']}>

```ts
const { users } = await xata.sql<UsersRecord>`SELECT * FROM \"Users\" ORDER BY properties->'address' DESC LIMIT 20`;
```

```python
records = xata.sql().query("SELECT * FROM \"Users\" ORDER BY properties->'address' DESC LIMIT 20;")
```

```jsonc
// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/sql

{
  "statement": "SELECT * FROM \"Users\" ORDER BY properties->'address' DESC LIMIT 20;"
}
```

</TabbedCode>

> ## Paginating Results

Reviewer: ```suggestion
## Paginating results
```

> This section doesn't apply to the TypeScript SDK, because it is offering a higher-level abstraction over the pagination mechanism. See the

Reviewer: ```suggestion
This section is not relevant for the TypeScript SDK, as it provides a more advanced abstraction layer for handling pagination. See the
```

Author: @joan-ing apologies I didn't see this edit before I merged. We can create a follow up PR to add this change.
